WebRoot/css/dwt.css

```diff
@@ -2041,7 +2039,9 @@ INPUT.xform_field {
 .ZSelectAutoSizingContainer					{	height:2rem; width:auto;	}		/* TODO: SIZE ASSUMPTION! */
 
 .ZSelect									{	@Select@		@SelectSpacing@			}
-.ZSelect .ZWidgetTitle						{	padding-left:5px; text-align:left;		}
+.ZSelect .ZWidgetTitle						{	@SelectTitle@		}
```

Comment:
Can't fix it right now with this commit. The file has usage of tabs which is causing this . In my opinion it might be good to handle this once & for all and change it to using spaces throughout .